test/test_parser.py

```diff
@@ -114,9 +114,14 @@ def process_docstring(lines): return _process_docstring(transform, lines)
                 if filter_clang_args is not None and root.get_clang_args() not in filter_clang_args:
                     continue
 
-                for docstrings in root.walk(recurse=False, filter_types=_filter_types(directive),
-                                            filter_names=_filter_names(directive)):
-                    for docstr in docstrings.walk(filter_names=_filter_members(directive)):
+                for primary in root:
+                    if _filter_types(directive) is not None and type(primary) not in _filter_types(directive):  # noqa: E501
```

Comment:
Nitpick: some of these lines are already mouthfuls, but these ones require an exception too. I may be biased though, I'm a stickler for 79 character limit :P

There's at least one more commit with a similar `E501` if you care to fix it.

Reply:
Thanks a lot for the review! I've got a bunch of follow-up changes lined up, including cleaning these up. So I think I'll just merge this.

I usually like to keep the lines under about a 100 chars or so, and I think that's what I've set up for flake8 config here. I think 80 is too restrictive. ;)